packages/protocol/test/TaikoTest.sol

```diff
@@ -42,7 +42,17 @@ import "../contracts/test/erc20/FreeMintERC20.sol";
 import "./DeployCapability.sol";
 import "./HelperContracts.sol";
 
-abstract contract TaikoTest is Test, DeployCapability {
+// For SGX remote attestation
```

Comment:
Do not add SGX specific settings in this file, you can create a TaikoSGXTest.sol which uses TaikoTest as its parent.